kbase-extension/static/kbase/js/util/jobLogViewer.js

```diff
@@ -986,9 +1152,13 @@ define([
                 key: {
                     type: 'job-log-deleted',
                 },
-                handle: function () {
+                handle: function (message) {
                     stopLogAutoFetch();
                     renderLog();
+                    awaitingLog = false;
```

Comment:
if the log has been deleted, we aren't going to have any joy when listening for it.